.github/workflows/build.yml

```diff
@@ -142,24 +142,38 @@ jobs:
 
             ## Install
 
+            #### macOS
+
             If you're on macOS, you can use Homebrew:
 
             ```bash
             $ brew install buildpacks/tap/pack
             ```
 
-            Otherwise:
-
-            1. Download the `.tgz` or `.zip` file for your platform
-            2. Extract the `pack` binary
-            3. (Optional) Add the directory containing `pack` to `PATH`, or copy `pack` to a directory like `/usr/local/bin`
+            #### Linux
+            
+            On Linux you can use the one of the following methods of installation.
+            
+            ##### Arch Linux
+            
+            - [pack-cli](https://aur.archlinux.org/packages/pack-cli/)
+            - [pack-cli-bin](https://aur.archlinux.org/packages/pack-cli-bin/)
             
-            On Linux you can use the following command:
+            ##### Command
```

Comment:
If I'm reading this right there are two paths here: arch and others (which I take to mean "other distros").
If that's right, I think the command should be nested under "Others".

Reply:
If we put command under others I would feel like it would be inconsistent since it's not applicable to Windows. Hope is to add other distros here soon.